src/go/k8s/tests/e2e/prestop-hook/00-cluster.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: redpanda.vectorized.io/v1alpha1
```

Comment:
I think it's kind of overkill to start new cluster just to verify the fields. I would say unit test is enough or just assert this in some other test that already boots up the cluster and maybe asserts other things. This would slow down test time and I think for very small gain

Reply:
makes sense. I'll replace it with a unit test